composer.json

```
@@ -29,7 +29,7 @@
         "symfony/dom-crawler": "^4.2",
         "symfony/console": "^4.2",
         "neos/composer-plugin": "^2.0",
-        "composer/composer": "^1.9",
+        "composer/composer": "^2.0",
```

```suggestion
        "composer/composer": "^1.9 || ^2.0",
```
If we don't want to strictly require ^2.0, we need to at least do this to allow PHP 8 builds to function.